dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/post-fee-record-correction.controller/index.ts

```diff
@@ -68,6 +76,8 @@ export const postFeeRecordCorrection = async (req: PostFeeRecordCorrectionReques
           },
         },
       });
+
+      await sendFeeRecordCorrectionRequestEmails(reasons, feeRecord.report.reportPeriod, feeRecord.exporter, feeRecord.report.bankId, user.email);
```

Comment:
if the email does not send, is the process technically complete?
If not, should we return a boolean from `sendFeeRecordCorrectionRequestEmails` and check if not true, then send an error code?

Reply:
If the email doesn't send sucessfully it will throw an error which gets caught in the try catch block in the controller and then we send the appropriate error response